### ISSUE TITLE
Consider refining or documenting the current range of opacities

### ISSUE THREAD

**Issue by jonathantneal** [RocketCommunicationsInc/astro-design-tokens#48]
The current opacity tokens represent a particularly narrow range of values; 35%, 40%, 45%, and 50%.

In the current web components library, the 40% value is represented as `--disabled-opacity`.

- Are all of these tokens used? Are any deprecated? Should any be removed?
- Where does the design system communicate these effects to users? Like:
  - As a consumer of the design system, how should I know which opacity reference is appropriate?
    - Like, is there a range of opacities because there is a range of disabled-ness-es?
    - Or, like, is there a range of opacities because specific colors look better with specific opacities?

Without understanding the effects of this particular range of opacities, I think it will be too difficult to use them effectively.

**Comment by markacianfrani** [comment]
Yep! Let's break out #2 into a separate discush. 

I think they should definitely be linting rules, but as an addition not in place of. It gets a little weird because design doesn't have a real concept of linting (yet 😉). So when we talk about "shipping" do we just mean the web output targets?

**Comment by jonathantneal** [comment]
Yes, for CSS the goal would be to only ship custom properties intended for use.

The bar is high for custom properties, because, unlike shipping stray blocks of HTML or JavaScript, which are rarely accessible to authors, shipping any stray custom property in CSS is automatically a publicly accessible feature.

**Comment by markacianfrani** [comment]
I should clarify--it's not that Reference tokens shouldn't ever be used/made available, rather,
System tokens are preferred. 

We prefer people use our context abstractions to prevent misuse (we want a consistent background color) But there's still going to be things we are missing. For example, what should the background color be if text is highlighted? There's no System guidance so I might grab a Reference yellow

**Comment by jonathantneal** [comment]
I like that, @markacianfrani. What do you think about, generally, expecting to ship reference tokens that represent scales? These would be things like sizing scales and color scales.

Example: [the 15 color scales in Radix](https://www.radix-ui.com/docs/colors/palette-composition/the-scales)

> For example, what should the background color be if text is highlighted? There's no System guidance so I might grab a Reference yellow

This is where a reference scales might be especially valuable. Consider this [yellow scale](https://www.radix-ui.com/docs/colors/palette-composition/the-scales#yellow). It enforces branding, protects contrast, and won’t bust the design if they switch to dark mode. (Note: the design system is currently limited to dark mode, where reasonably and naturally reaching for a reference yellow that is only absolute yellow might be make marked text illegible.)
